util/pom.xml

```diff
@@ -70,8 +70,8 @@
         </dependency>
 
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
```

Comment:
In the same ticket, we should rely on logback-classic in tests...

Reply:
See https://issues.apache.org/jira/browse/JSIEVE-112

Reply:
alright will do in a separate PR